test/lib/ansible_test/_util/target/setup/bootstrap.sh

```diff
@@ -337,16 +337,6 @@ bootstrap_remote_rhel_9()
             ${py_pkg_prefix}-cryptography
             ${py_pkg_prefix}-pyyaml
             "
-
-        # The following OS packages are missing for 3.11 (and possibly later) so we just
-        # skip them and let ansible-test install them from PyPI.
```

Comment:
It would be good to keep some kind of comment that explains why `packaging` and `resolvelib` are not using OS packages. I think some variation of this comment can be included just below the one above about Jinja2.